### DISCUSSION TITLE
Tesla custom integration token

### DISCUSSION THREAD

**Discussion by sagara939** [alandtse/tesla#910] [Q&A]
Hi all,

today this integration stop working and i can't generate new token with the app "telsa tokens". 
How i can do this?

THANKS!

**Comment by thomaswwp** [comment]
https://tesla-info.com/tesla-token.php

**Reply by thomaswwp** [reply]
I too cold not find Tesla Tokens in the google app store but this site works well. You just have to tust they are not a bad actor. 🙏

**Reply by sagara939** [reply]
thanks, i see this page. but the tokens i think it work for about 8 hours. and, the application need a proxy settings. where i can found this settings? 
thanks!

**Reply by thomaswwp** [reply]
Sorry, guess my car is old enough that the refresh token was enough.
